docs/architecture/governance/improvement-process.md

```diff
@@ -5,49 +5,54 @@ last_update:
 ---
 As Tezos is a large decentralized project that is constantly evolving, its members need to have a mechanism for proposing improvements to the ecosystem.
 
-A Tezos Improvement Proposal (or TZIP, pronounced "tee-zip") is a document that offers ways to improve Tezos via new features, tools, or standards (e.g. smart contract specifications).
+A Tezos Improvement Proposal (or TZIP, pronounced "tee-zip") is a document that offers ways to improve Tezos via new features, tools, or standards (e.g. smart contract interface specifications).
 
-Specifically, a TZIP is a design document providing information to the Tezos community, describing a feature for Tezos or its processes or environment, and supporting the formal protocol governance process. The TZIP process complements (but is subsidiary to) Tezos' formal on-chain governance process.
+Specifically, a TZIP is a design document proposed to the Tezos community, describing a feature for Tezos or for the related processes, tools, or artefacts (e.g., smart contracts).
 
-The TZIP document should contain a concise technical specification and rationale that clearly articulates what the proposal is, how it may be implemented, and why the proposal is an improvement. It should also include an FAQ that documents, compares, and answers alternative options, opinions, and objections.
+TZIPs address a different need when compared to the formal online governance process for protocol amendments.
+Typically, TZIPs do not impact the Tezos protocol at all. They rather concern other parts of the Tezos ecosystem.
 
-An explorer for all of the TZIPs, both past and present, can be found [here](https://tzip.tezosagora.org/).
+In theory, a TZIP could support the formal online governance process by defining a specification including some required protocol change, that can later be implemented in a protocol amendment. In such an eventuality, the TZIP process would complement (but remain subsidiary to) Tezos' formal on-chain governance process.
```

Comment:
Is this hypothetical important enough to include here?

Reply:
I think the statement about this potential use of TZIP is confusing because it says how TZIP could be part of the formal governance process, but the previous paragraph says that they are not part of the formal process. Maybe a better example would be that they create token standards that are not required but something that the community agrees on.